variables.mk

```diff
@@ -22,7 +22,7 @@ TAG              := ${FULL_VERSION}
 BRANCH           := $(shell git rev-parse --abbrev-ref HEAD)
 
 ### third-party components version
-CSI_PROVISIONER_TAG := v1.6.0
```

Comment:
Old tag is still in use, right? https://github.com/dell/csi-baremetal-operator/blob/21a6f06652587b1e426ff9094f3f6b1d56882bd3/charts/csi-baremetal-deployment/templates/csi-baremetal_v1_deployment.yaml#L45

Reply:
Yes, but for out kind version 1.18 it's ok to use provisioner version 3.0.

Reply:
Do you mean the provisioner 3.0 will work with k8s 1.18 as well?

Reply:
Yes, it can. v1 api version of CSINode crd was introduced in k8s 1.17

Reply:
In this case why do we need to have k8s version check here - https://github.com/dell/csi-baremetal-operator/blob/master/charts/csi-baremetal-deployment/templates/csi-baremetal_v1_deployment.yaml#L42 and deprecate tag here - https://github.com/dell/csi-baremetal-operator/blob/master/charts/csi-baremetal-deployment/values.yaml#L83 if minimal supported version is 1.18?

Reply:
If we do not support the earlier versions - than there is no need in it